Makefile

```diff
@@ -87,28 +87,34 @@ prepare-settings:
 	$(call message,$(PROJECT_NAME): Making settings.php writable)
 	$(call docker-wodby, php chmod 666 web/sites/default/settings.php)
 
+prepare-git:
+	$(call message,$(PROJECT_NAME): Setting up git hooks)
+	git config core.hooksPath .git-hooks
+	ln -s $(shell pwd)/.git-hooks/* $(shell pwd)/.git/hooks/
```

Comment:
Why do we do both symbolic linking of hooks into `.git/hooks` path and changing the `core.hooksPath` to the `.git-hooks` dir we are using to create symbolic links from? It looks like only one of these things is needed.